v3/util/misc.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+func AppendToStringSemicolonDelim(this *string, s string) {
```

Comment:
This function is very specific to what is trying to be accomplished in `GetSubjectElement`. You would mind simply placing it in-line in that function?

Reply:
Moved this function to qc_stmt because it used extensively in the qc context. I hope this is ok.